examples/pbam/1a63.in

```diff
@@ -2,19 +2,23 @@ read
     mol pqr 1a63.pqr
 end
 elec name comp_solv        # Toy charges
-    pbam
+    pbam-auto
 
     runtype electrostatics                           # Can be energyforce, electrostatics etc
     runname elec_1a63                                # Output name for files created by program
     temp 298.15                                      # System temperature (K)
     pdie 2.0                                         # Solute dielectric                 
     sdie 80.0                                        # Solvent dielectric                
-    ion charge 0.0 conc 0.10 radius 0.0              # Monovalent salt conc in M
+    # WARN: ion is no longer supported in pbam-auto
+    # ion charge 0.0 conc 0.10 radius 0.0              # Monovalent salt conc in M
 
-    dime 110
+    # WARN: dime is no longer supported in pbam-auto
+    # dime 110 110 110
 
-    write pot dx 1a63_0.1M.dx
-    write 3dmap  1a63_0.1M.map
+    # WARN: write is no longer supported in pbam-auto
+    # write pot dx 1a63_0.1M.dx
```

Comment:
Really?  Why is `write` not supported for this?  Was it never supported?

Reply:
It may be supported but I was going off of the [pbam-auto](https://apbs.readthedocs.io/en/latest/using/input/elec/pbam-auto.html) spec.

Reply:
Got it -- looks like `3dmap` implements the same functionality.